src/Ydb.php

```diff
@@ -173,6 +179,14 @@ public function meta(): array
         return $meta;
     }
 
+    public function opts(): array
+    {
+        $opts = $this->opts;
+        $opts['credentials'] = $this->iam()->getCredentials();
```

Comment:
At the place need to clone original dict `$this->opts` and change copy or move `$opts['credentials'] = $this->iam()->getCredentials();` to initialization phase for prevent unexpected change the dict.